### PR TITLE
Refactor: App 648 - Connect Explore to SDK

### PR DESCRIPTION
**Pull request by WissenIstNacht**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task: [648](https://aragonassociation.atlassian.net/browse/APP-648)

## Type of change

- [x] Refactoring
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: packages/web-app/src/hooks/useDaos.tsx [hunk 1]
```diff
@@ -1,64 +1,62 @@
...
-type DaoOverview = {
-  name: string;
-  description: string;
-  logo?: string;
-  chainId: number;
-  daoType: DaoType;
-};
+// NOTE Currently the DAOs are fetched using the SDK's getDaos method. This
+// method simply returns a list of mocked DAOs. In particular, it does not allow
+// to pass any arguments like "newest" or "popular". Consequently the fetched
```
Comment (Fabricevladimir) [inline packages/web-app/src/hooks/useDaos.tsx]: Params object has now been implemented for the daos. It allows paginated queries as well as filters like created at and popular. You can check it out [here](https://github.com/aragon/sdk/blob/d9514951c31fdd4510d870845796186ba4ae48da/javascript/modules/client/src/client.ts#L172)

**Review by Fabricevladimir — COMMENTED** [review]
Thanks Vali! Left a comment about new SDK additions

**Review by sepehr2github — APPROVED** [review]
thanks Vali, Just left a minor comment

**Comment by WissenIstNacht** [comment]
@sepehr2github Are your sure? I can't see the comment 🙈

File: packages/web-app/src/hooks/useDaos.tsx [hunk 2]
```diff
@@ -1,64 +1,55 @@
...
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. In blandit enim ac quam porta tempus. Morbi feugiat leo in ultricies commodo. Praesent tempus neque eu tellus pulvinar, suscipit imperdiet erat laoreet. Vivamus interdum risus fermentum magna convallis tristique. Praesent sit amet venenatis nulla, non ornare lectus. Quisque elit tortor, suscipit sed mi id, mattis tempus felis. Praesent bibendum viverra auctor. Cras finibus, mauris at congue cursus, nisl magna semper lorem, quis ornare odio sem id nulla. Vestibulum fermentum commodo tortor, ac vehicula libero. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Nunc facilisis nisl viverra, fermentum dui non, ultricies dolor. Mauris ornare varius est, eu finibus tellus lobortis quis. Nullam sagittis vulputate mi in tincidunt. Nam tempor lacus lorem, ac consectetur velit malesuada sed. ',
-        chainId: 4,
-        daoType: 'wallet-based',
-      });
+      const sortParam =
+        useCase === 'popular' ? DaoSortBy.POPULARITY : DaoSortBy.CREATED_AT;
+      const daoDetails =
+        (await client?.methods.getDaos({sortBy: sortParam, limit: count})) ||
```
Comment (Fabricevladimir) [inline packages/web-app/src/hooks/useDaos.tsx:41]: I'm curious as to why the empty array when `data` is initialized with an empty one already?

**Review by Fabricevladimir — APPROVED** [review]

File: packages/web-app/src/hooks/useDaos.tsx (hunk 2 above)
Comment (WissenIstNacht) [reply packages/web-app/src/hooks/useDaos.tsx:41]: Ahm `getDaos` could return `undefined`. But the state hook doesn't do `undefined`.

Obviously we could change the state to accept `undefined`, but I think that's not the right way to go. Firstly, the fewer `undefined` and `null`, the better. Secondly, you can still do `map`, `forEach`, etc. on an empty array without having to guard against undefined values.

**Pull request merged**